db/system_keyspace.cc

```diff
@@ -1463,7 +1463,47 @@ struct local_cache {
     system_keyspace::bootstrap_state _state;
 };
 
+future<> system_keyspace::peers_table_read_fixup() {
+    static bool done = false;
+    if (done) {
+        co_return;
+    }
+    done = true;
+
+    const auto cql = format("SELECT peer, host_id, WRITETIME(host_id) as ts from system.{}", PEERS);
+    std::unordered_map<utils::UUID, std::pair<net::inet_address, int64_t>> map{};
+    const auto cql_result = co_await execute_cql(cql);
+    for (const auto& row : *cql_result) {
+        const auto peer = row.get_as<net::inet_address>("peer");
+        if (!row.has("host_id")) {
+            slogger.warn("Peer {} has no host_id in system.{}, the record is broken, removing it",
```

Comment:
I'll merge, but this should be reported as an ERROR -- it most likely indicates a bug.
Please send a follow-up to address this

Reply:
done for both problems - missing `host_id` and duplicate `IP`-s.